os/binfmt/libelf/libelf_addrenv.c

```diff
@@ -151,9 +151,15 @@ int elf_addrenv_alloc(FAR struct elf_loadinfo_s *loadinfo, size_t textsize, size
 #ifdef CONFIG_APP_BINARY_SEPARATION
 #ifdef CONFIG_DEBUG_MM_HEAPINFO
 	ARCH_GET_RET_ADDRESS
-	loadinfo->textalloc = (uintptr_t)mm_malloc(loadinfo->uheap, textsize + datasize, retaddr);
+	loadinfo->textalloc = (uintptr_t)mm_memalign(loadinfo->uheap, textsize, textsize, retaddr);
+	ARCH_GET_RET_ADDRESS
```

Comment:
This macro sets `retaddr` and it is not changed from mm_xxx.
So I think we don't need to call again.

Reply:
Ok. Will remove the duplicates.